recipe/meta.yaml

```
@@ -33,10 +33,10 @@ requirements:
     - typing_extensions
 
 test:
-  imports:
-    - datafusion
   commands:
     - pip check
+    - python -c 'from importlib.util import find_spec; assert find_spec("datafusion")'
+    - python -c 'import datafusion'                                                     # [build_platform == target_platform]
```

Appears Windows didn't like the single quotes outside with double quotes within. So swap them

```suggestion
    - python -c "from importlib.util import find_spec; assert find_spec('datafusion')"
    - python -c "import datafusion"                                                     # [build_platform == target_platform]
```